subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junitplatform/JUnitPlatformTestFramework.java

```diff
@@ -84,6 +85,11 @@ public TestFrameworkDetector getDetector() {
         return null;
     }
 
+    @Override
+    public void close() throws IOException {
+
```

Comment:
I'm curious why JUnit and TestNG maintain some state here, but Jupiter does not require it. Maybe worth noting in a comment?

Reply:
done